x-pack/plugins/cases/public/containers/mock.ts

```diff
@@ -1158,6 +1158,18 @@ export const customFieldsMock: CaseUICustomField[] = [
 ];
 
 export const customFieldsConfigurationMock: CasesConfigurationUICustomField[] = [
-  { type: CustomFieldTypes.TEXT, key: 'test_key_1', label: 'My test label 1', required: true },
-  { type: CustomFieldTypes.TOGGLE, key: 'test_key_2', label: 'My test label 2', required: false },
+  {
+    type: CustomFieldTypes.TEXT,
+    key: 'test_key_1',
+    label: 'My test label 1',
+    required: true,
+    defaultValue: 'My default value',
+  },
+  {
```

Comment:
Let's add one more that will contain a non-required field as before. We want to be sure that optional fields are working as expected.